pkg/mapper/mapper_test.go

```diff
@@ -491,6 +517,30 @@ mappings:
     `,
 			configBad: true,
 		},
+		// Config with multiple explicit metric types
```

Comment:
Where my new test actually starts. Just adding a comment to increase visibility.